draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1514,86 +1514,70 @@ still provide adequate output for incomplete logs.
-handling. However, in many cases the largest risk factors arise from data that
-can be considered as potenially-sensitive or sensitive.
+qlog operators and implementers need to consider security and privacy risks when
+handling qlog data, including logging, storage, usage, and more. Data can be
+categorized as non-sensitive, potentially-sensitive, or sensitive. The
+considerations presented in this section may pose varying risks depending on the
+the data itself or its handling. Generally, the largest risk factors arise from
+data considered as potenially-sensitive or sensitive.
 
-The following is a non-exhaustive list of such fields and types of data that can
-be carried in qlog data:
+The following is a non-exhaustive list of example data types could contain
+sensitive information that might allow identification or correlatation of
+individual connections, endpoints, users or sessions across qlog or other data
+sources (e.g., captures of encrypted packets):
 
-* IP addresses and transport protocol port numbers, which can be used to
-  uniquely identify individual connections, endpoints, and potentially users.
+* IP addresses and transport protocol port numbers.
 
-* Session, Connection, or User identifiers which can be used to correlate
-  nominally separate contexts. For example, QUIC Connection IDs can be used to
-  identify and track users across geographical networks {{Section 9.5 of
-  !RFC9000}}).
+* Session, Connection, or User identifiers e.g., QUIC Connection IDs {{Section
+  9.5 of !RFC9000}}).
 
-* System-level information such as CPU, process, or thread identifiers.
+* System-level information e.g., CPU, process, or thread identifiers.
 
-* Stored State which can be used to correlate individual connections or sessions
-  over time. Examples include QUIC address validation and retry tokens, TLS
-  session tickets, and HTTP cookies.
+* Stored State e.g., QUIC address validation and retry tokens, TLS session
+  tickets, and HTTP cookies.
 
-* Decryption keys, passwords, and tokens which can be used with other data
-  sources (e.g., captures of encrypted packets) to correlate qlog data to a
-  specific connection or user or leak additional information. Examples include
-  TLS decryption keys and HTTP-level API access or authorization tokens.
+* TLS decryption keys, passwords, and HTTP-level API access or authorization tokens.
 
-* Data that can be used to correlate qlogs to other data sources (e.g., captures
-  of encrypted packets). Examples include high-resolution event timestamps or
-  inter-event timings, event counts, packet and frame sizes.
+* High-resolution event timestamps or inter-event timings, event counts, packet
+  sizes, and frame sizes.
 
-* Full or partial encrypted raw packet and frame payloads, which can be used
-  with other data sources (e.g., captures of encrypted packets) to correlate
-  qlog data to a specific connection or session.
+* Full, or partial encrypted, raw packet and frame payloads.
```

Comment:
```suggestion
* Full or partial raw packet and frame payloads that are encrypted.
```

@rmarx is something like this the intent?

Reply:
In combination with the consistency changes below for plaintext, this indeed seems ok